wger/nutrition/static/js/comparison.js

```diff
@@ -0,0 +1,127 @@
+/*
+ This file is part of wger Workout Manager.
+
+ wger Workout Manager is free software: you can redistribute it and/or modify
+ it under the terms of the GNU Affero General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ wger Workout Manager is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU Affero General Public License
+ */
+
+function modifyTimePeriod(data, pastNumberDays) {
+  var filtered;
+  var date;
+  if (data.length) {
+    if (pastNumberDays !== 'all') {
+      date = new Date();
+      date.setDate(date.getDate() - pastNumberDays);
+      filtered = MG.clone(data).filter(function (value) {
+        return value.date >= date;
+      });
+      return filtered;
+    }
+  }
+  return data;
+}
+
+function compareUser(username) {
+  var url;
+  var chartParams;
+  var nutritionChart;
```

Comment:
We can assign this `var` on declaration i.e `var nutitionChart = {}`. Same goes for `chartParams` and `url`.